### PR TITLE
Added Logout and Settings options on profile page

### PR DESCRIPTION
**Pull request by sumitkr2000**
Hey, @narayan954 I have successfully added the logout and setting options on the profile page and removed these options from the navbar. Please review my pr and merge it. Thank you!

- Issue no. #675 

image

### CODE REVIEW

**Comment by narayan954** [comment]
As I can see, you made a mistake of creating a branch from a feature branch instead of the master branch, hence the commits of that feature branch are also reflected on this branch. I can accept this pr by squashing it so that this won't reflect the commit history. Please look after this mistake from the next time.

**Comment by narayan954** [comment]
when you use `git checkout` to create a branch, make sure you're checking out from updated master branch. For this you may first checkout to master, update it and then create a new branch. This helps in having a clean commit history

**Review by narayan954 — CHANGES_REQUESTED** [review]
logout confirmation seems to be missing, can you fix that please?

**Comment by sumitkr2000** [comment]
> As I can see, you made a mistake of creating a branch from a feature branch instead of the master branch, hence the commits of that feature branch are also reflected on this branch. I can accept this pr by squashing it so that this won't reflect the commit history. Please look after this mistake from the next time.

Okay, sorry bro for the mistake. I will not repeat this mistake from next time.

**Comment by sumitkr2000** [comment]
> logout confirmation seems to be missing, can you fix that please?

Yeah sure please give me some time

**Comment by sumitkr2000** [comment]
@narayan954 I have added the logout modal now and it's working fine. Please review this pr.

File: src/pages/Profile/index.jsx [hunk 1]
```diff
@@ -19,16 +18,57 @@ import {
...
 import ViewsCounter from "./views";
 import firebase from "firebase/compat/app";
+import logo from "../../assets/logo.webp";
+import { makeStyles } from "@mui/styles";
 import { useSnackbar } from "notistack";
 
 const Post = lazy(() => import("../../components/Post"));
 const SideBar = lazy(() => import("../../components/SideBar"));
 
+export function getModalStyle() {
```
Comment (narayan954) [inline src/pages/Profile/index.jsx]: why were you adding this function again if you could just import it?

File: src/pages/Profile/index.jsx [hunk 2]
```diff
@@ -19,16 +18,57 @@ import {
...
+    // left: `${left}%`,
+    transform: `translate(-${top}%, -50%)`,
+    padding: `${padding}%`,
+    borderRadius: `${radius}%`,
+    textAlign: "center",
+    backgroundColor: "var(--bg-color)",
+  };
+}
+
+export const useStyles = makeStyles((theme) => ({
```
Comment (narayan954) [inline src/pages/Profile/index.jsx]: same repeat

**Review by narayan954 — APPROVED** [review]
I'm removing the duplicate code, do let me know if there was any special reason to duplicate it

File: src/pages/Profile/index.jsx (hunk 1 above)
Comment (sumitkr2000) [reply src/pages/Profile/index.jsx]: I removed it from app.js and then pasted it to the profile page. Later I realized that it is getting used somewhere else also. So forgot to remove it from Profile.js.

**Comment by sumitkr2000** [comment]
Thanks!

**Pull request merged**